examples/persistentconversationbot.py

```diff
@@ -77,18 +77,19 @@ def regular_choice(update: Update, context: CallbackContext) -> int:
     context.user_data['choice'] = text
     if context.user_data.get(text):
         reply_text = (
-            f'Your {text}, I already know the following about that: {context.user_data[text]}'
+            f'Your {text}? I already know the following about that: {context.user_data[text]}'
         )
     else:
         reply_text = f'Your {text}? Yes, I would love to hear about that!'
-    update.message.reply_text(reply_text)
+    update.message.reply_text(reply_text, reply_markup=ForceReply(selective=True))
```

Comment:
Mh, if people ask for examples on how to use `ForceReply` I feel like it's not a good idea to have them run a conversationbot … Could you mind moving this to `echobot.py`? You can have the `start` message mention the user, which should work with `selective=True`

Reply:
Alright. I included this here cause it felt the like the most natural place to put it.